apps/condo/domains/notification/constants/constants.js

```diff
@@ -128,10 +132,17 @@ const MESSAGE_PROCESSING_STATUS = 'processing'
 const MESSAGE_ERROR_STATUS = 'error'
 const MESSAGE_DELIVERED_STATUS = 'delivered'
 const MESSAGE_CANCELED_STATUS = 'canceled'
+const MESSAGE_SENT_STATUS = 'sent'
+const MESSAGE_READ_STATUS = 'read'
```

Comment:
it's not a status. It's another field. Status is about a delivery process. The read is about a user's action. It's bad idea to mix it in one field